corehq/apps/app_manager/util.py

```diff
@@ -572,3 +572,11 @@ def get_form_data(domain, app):
         module_meta['forms'] = forms
         modules.append(module_meta)
     return modules, errors
+
+
+def sort_nodeset_columns_for_detail(detail_type, detail):
```

Comment:
Could this be a method on the detail class? It's pretty specific to be in a utils file.